lib/auth.ts

```diff
@@ -1,12 +1,26 @@
 import * as ClientOAuth2 from "client-oauth2";
 import { sha256 } from "js-sha256";
 import { btoa } from "abab";
-import { ClientOpts, GetTokenOpts, GetAuthUriOpts } from "./types";
+import {
+  ClientOpts,
+  GetAuthUriOpts,
+  Challenge,
+  ChallengeStatus,
+  HttpMethod
+} from "./types";
+import "cross-fetch/polyfill";
```

Comment:
This is a dependency of `graphql-request` that we already use. I think it's best to not include another lib and stick to that one, any opinion?

Reply:
I agree